model/resolver/resolver.go

```diff
@@ -609,3 +602,21 @@ clusterRoleLoop:
 	}
 	return errors
 }
+
+// calculateConfigurationTemplates caculates the global configuration templates
```

Comment:
nit: `caculates` vs `calculates`

Reply:
Fixed in 311c0c12dbf55e61bd2f9dc2332075f7a8afc131